### PR TITLE
feat: add use cases and implementation considerations

### PR DESCRIPTION
**Pull request by tplooker**
Adds two usecases for this specification and some implementation considerations.

### CODE REVIEW

**Comment by tplooker** [comment]
Ping @tlodderstedt @awoie @Sakurann

**Review by awoie — APPROVED** [review]
LGTM

File: spec.md [hunk 1]
```diff
@@ -247,6 +247,26 @@ TODO
...
+
```
Comment (Sakurann) [inline spec.md:263]:
```suggest

## Personal Data Stores

_An end-user providing information from a server-based personal data store_

Tom stores his personal data such as claims about himself in a personal data store (PDS) hosted on a server. He wants to authenticate himself to the Relying Party using the claims stored in PDS. Upon Tom's consent, PDS issues authorization code to the RP that RP can exchange with the claims that Tom intended to share.
```

Comment (tplooker) [reply spec.md:263]: Whilst I like this usecase description I dont think it fits within the narrative / scope of portable identities? Perhaps the usecase of being able to change the personal data store provider would be a better fit?

Comment (tplooker) [reply spec.md:263]: @Sakurann can you perhaps offer this usecase in a subsequent PR?

**Pull request merged**